doc/src/appendices/suiterc-config-ref.rst

```diff
@@ -2033,9 +2042,10 @@ earlier in the script.  See also :ref:`TaskExecutionEnvironment`.
 Replace ``__VARIABLE__`` with any number of environment variable
 assignment expressions. Order of definition is preserved so values can
 refer to previously defined variables. Values are passed through to the task
-job script without evaluation or manipulation by cylc, so any variable assignment
-expression that is legal in the job submission shell can be used.
-White space around the ``=`` is allowed (as far as cylc's suite.rc
+job script without evaluation or manipulation by cylc (with the exception of
```

Comment:
```suggestion
job script without evaluation or manipulation by Cylc (with the exception of
```

Cylc was once deliberately lowercased, but not anymore (but you can leave these for a later global fix if you like).